src/prefect/executors/dask.py

```diff
@@ -614,14 +616,24 @@ def cull2(dsk, keys):  # type: ignore
                 finally:
                     dask.multiprocessing.cull = cull
 
-            config = {"optimization.fuse.active": False}
+            config["optimization.fuse.active"] = False
         else:
-            config = {}
 
             @contextmanager
             def patch() -> Iterator[None]:
                 yield
 
+        # Patch around https://github.com/PrefectHQ/prefect/issues/4537
```

Comment:
any reason not to slide this up into the previous `if` block?

Reply:
I want to maintain these "patches" separately since they are focusing on fixing very different upstream issues. This will make the diff easier to parse when we remove one.